packages/legend-extension-dsl-data-space/src/stores/query/DataSpaceQueryEditorStore.ts

```diff
@@ -150,6 +150,11 @@ export class DataSpaceQueryEditorStore extends QueryEditorStore {
         this.queryBuilderState.initialize(
           this.queryBuilderState.graphManagerState.graphManager.createDefaultBasicRawLambda(),
         );
+        await Promise.all([
```

Comment:
See my comment in `QueryBuilderChangeDetectionState`, we should only need to do:
`await queryBuilderState.changeDetectionState.start();` here